tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import PageRegion
+
+
+class BottomBar(PageRegion):
+    _bottom_bar_locator = (By.ID, 'bottombar')
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
+    def __init__(self, marionette):
+        PageRegion.__init__(self, marionette, self._bottom_bar_locator)
+        
+        view = self.marionette.find_element(*self._bottom_bar_locator)
+        bottom_bar_position = view.rect
+
+        Wait(self.marionette).until(lambda m: int(view.location['y']) == int(bottom_bar_position["y"])and view.is_displayed())
```

Comment:
Ok, so do this before the PageRegion.__init__( call, then call:
`PageRegion.__init__(self, marionette, view)`
(although I would call it root)